orchestration/docker.go

```diff
@@ -152,7 +175,7 @@ func configureChainSpec(nodeType string, network string) {
 	networkEnv := dockerExec("cat", networkEnvPath)
 	networkEnvMap, err := godotenv.Unmarshal(networkEnv)
 	if err != nil {
-		exitWithError("Error unmarshalling network env:", err)
+		log.Fatal(err)
```

Comment:
Not sure yet where you plan to use configureChainSpec, but we should probably just return the errors and leave `log.Fatal` to the cobra commands